ncm-metaconfig/src/main/metaconfig/slurm/pan/schema.pan

```diff
@@ -95,9 +101,11 @@ type slurm_scheduler_parameters = {
     'enable_user_top' ? boolean
     'Ignore_NUMA' ? boolean
     'inventory_interval' ? long(0..)
-    '{kill_invalid_depend}' ? boolean
+    '{kill_invalid_depend}' ? boolean with {
+        deprecated(0, 'kill_invalid_depend has moved to DependencyParameters'); true;
+    }
     'max_array_tasks' ? long(0..)   # should be smaller than MaxArraySize
-    'max_depend_depth' ? long(0..)
+    'max_depend_depth' ? long(0..) with {deprecated(0, 'max_depend_depth has moved to DependencyParameters'); true; }
```

Comment:
this is not escaped, so won't work anyway

Reply:
fixed